backend/src/Resolvers.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+    Query: {
+        posts: (parent, args, {dataSources}) => dataSources.postsDataStore.allPosts(),
+        users: (parent, args, {dataSources}) => dataSources.userDataStore.allUsers()
```

Comment:
`Query: {
     users: async (_parent, _args, context) => {
       return context.dataSources.db.users();
     },
     posts: async (_parent, _args, context) => {
       return context.dataSources.db.posts();
     },`
you can make it a bit more elegant by defining DataSource and Context above and formulating method here. But you also wrote very nicely!